core/scheduler/scheduler.go

```diff
@@ -319,11 +319,11 @@ func (s *Scheduler) resolveAttDuties(ctx context.Context, slot core.Slot, vals v
 			continue
 		}
 
-		duty := core.NewAttesterDuty(int64(attDuty.Slot))
+		duty := core.NewAttesterDuty(uint64(attDuty.Slot))
```

Comment:
Can `attDuty.Slot` be moved to uint64 as well?

Reply:
`attDuty.Slot` is of `eth2p0.Slot` type. Since, `NewAttesterDuty(slot uint64)`, we need to typecast the eth2p0.Slot to uint64 type